relationalai-sdk-snowflake/README.md

```diff
@@ -0,0 +1,81 @@
+# The RelationalAI Software Development Kit for Snowflake Snowpark Python
+
+The RelationalAI (RAI) SDK for Python enables developers to access the RAI
+APIs from Snowflake Snowpark.
+
+* You can find RelationalAI product documentation at <https://docs.relational.ai>
+* You can find RelationalAI product documentation for Snowflake integration at <https://docs.relational.ai/preview/snowflake>
+* You can learn more about RelationalAI at <https://relational.ai>
+
+## Getting started
+
+### Requirements
+
+* Python 3.7+
```

Comment:
says >=3.8 in the toml file

Reply:
Right, changed it to 3.8 last minute since snowflake-snowpark-python supports >=3.8. Will change the README.